src/documentToNodes.js

```diff
@@ -1,18 +1,35 @@
-import * as R from 'ramda'
-import * as RA from 'ramda-adjunct'
+import {
+  compose,
+  cond,
+  test,
+  always,
+  T,
+  identity,
+  prop,
+  find,
+  propEq,
+  pick,
+  then,
+  fromPairs,
+  map,
+  toPairs,
+  omit,
+  propOr,
+} from 'ramda'
```

Comment:
Can't remember… did we add the Ramda Babel plugin? If yes, we can go back to `import * as R from 'ramda'`. Can keep it this way if we didn't put the plugin though. Not a big deal

Reply:
We did! I didn't realize that `import R from 'ramda'` still worked with the `babel` plugin. Too used to the `lodash` one. This has been changed.